src/App.svelte

```diff
@@ -1,9 +1,13 @@
 <script>
   import svelteLogo from './assets/svelte.svg';
+  import Router from 'svelte-spa-router';
+  import { routes } from './routes/index.js';
   import { Badge } from 'flowbite-svelte';
 </script>
 
+<Router {routes} />
+
 <main>
   <Badge>App</Badge>
-  <img src={svelteLogo} alt="Svelte Logo" />
+  <img src="{svelteLogo}" alt="Svelte Logo" />
```

Comment:
Do we need to parentheses  brackets here?

Reply:
> Do we need to parentheses brackets here?

Its eslint issue will be fixed in the my next PR